src/modules/graphics/wrap_GraphicsShader.lua

```diff
@@ -54,7 +54,8 @@ GLSL.SYNTAX = [[
 	#define DepthCubeImage samplerCubeShadow
 #endif
 #define extern uniform
-#ifdef GL_EXT_texture_array
+#if defined(GL_EXT_texture_array) && (!defined(GL_ES) || __VERSION__ > 100)
+#define texture_arrays_enabled
```

Comment:
I am not sure how correct this is. `(!defined(GL_ES) || __VERSION__ > 100)` is my attempt to say "if we're on OpenGL ES 2.0". It works on my Ouya, but I'm not sure if it does what I want elsewhere.

I'm also not sure if the `#define texture_arrays_enabled` style is a good idea.

Feedback is welcome.

Reply:
This will need some tweaks to work with how LÖVE does things with OpenGL ES, but the reasoning and changes needed are so obscure that I think I'll just merge this as-is and make the changes after.

Reply:
Thanks!